### PR TITLE
Migrate fileset to ECS

### PR DESCRIPTION
**Pull request by ruflin**
The following fields were migrated to ECS:

* fileset.name -> event.dataset
* fileset.module -> event.module

Changes:

* Update generated files
* Update tests
* Update changelog
* Update migration file

### CODE REVIEW

**Comment by ruflin** [comment]
Test failure is not related.

File: dev-tools/ecs-migration.yml [hunk 1]
```diff
@@ -11,7 +11,17 @@
 #   # Copy to is useful for fields where multiple fields map to the same ECS field
 #   copy_to: true-if-field-should-be-copied-to-target-in-6x
```
Comment (ph) [inline dev-tools/ecs-migration.yml]: <3 this file

File: filebeat/tests/system/test_modules.py [hunk 2]
```diff
@@ -176,7 +176,7 @@ def _test_expected_events(self, test_file, objects):
                 clean_keys(obj)
 
                 # Remove timestamp for comparison where timestamp is not part of the log line
-                if obj["fileset.module"] == "icinga" and obj["fileset.name"] == "startup":
+                if obj["event.module"] == "icinga" and obj["event.dataset"] == "startup":
```
Comment (ph) [inline filebeat/tests/system/test_modules.py]: I've seen that hack too, I think we could add a test.json inside the module to configure the assertions, to do in another PR of course.

Comment (ruflin) [reply filebeat/tests/system/test_modules.py]: yes, would be nice to do it differently.

**Comment by ruflin** [comment]
@ph For the dashboards: Yes plan is to hopefully be able to do it in an automated way.
